app/Models/QuestionOption.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class QuestionOption extends Model
+{
+    use HasFactory;
+    
+    protected $fillable = [
+       
+        'question_id',
+        'option',
+    ];
+
+    /**
+     * Returns the question which this questionOptions belongs to
+     * 
+     * @Return belongsTo
+     * 
+     */
+    public function question()
```

Comment:
```suggestion
    public function question(): BelongsTo
```
Maybe you can add al the return options like this in you classes?